src/main/java/org/folio/circulation/resources/RequestFromRepresentationService.java

```diff
@@ -99,6 +103,9 @@
         Request::truncateRequestExpirationDateToTheEndOfTheDay))
       .thenComposeAsync(r -> r.after(when(
         this::shouldFetchInstance, this::fetchInstance, req -> ofAsync(() -> req))))
+      .thenComposeAsync(r -> r.after(when(
+        this::shouldFetchInstance, this::fetchItemsOfInstanceForTheRequest, req -> ofAsync(() -> req))))
```

Comment:
It looks like we are going to fetch all the items of an instance even if we have Item Level Request, which does not seem right to me. Maybe we should a condition and rename a method to fetchItemsOfInstanceForTheTlr?

Reply:
fixed